ts/features/lollipop/store/index.ts

```diff
@@ -8,11 +8,61 @@ import {
   PersistPartial,
   persistReducer
 } from "redux-persist";
+import { pipe } from "fp-ts/lib/function";
 import { Action } from "../../../store/actions/types";
 import { isDevEnv } from "../../../utils/environment";
 import lollipopReducer, { LollipopState } from "./reducers/lollipop";
 
-export const CURRENT_REDUX_LOLLIPOP_STORE_VERSION = 0;
+export const CURRENT_REDUX_LOLLIPOP_STORE_VERSION = 1;
+
+export const migrationKeyTag = (
```

Comment:
Apart from tests this function seems not to be used anywhere, do we actually need it?

Reply:
let's move it to test file then 😉